jwt.go

```diff
@@ -428,9 +428,21 @@ func stringify(val interface{}) string {
 		return stringer.String()
 	}
 
+	if slice, ok := val.([]interface{}); ok {
+		return stringify_slice(slice)
+	}
+
 	return ""
 }
 
+func stringify_slice(slice []interface{}) string {
```

Comment:
Please use camelcase naming convention, `stringifySlice` should be used.

Reply:
- [X] use camelCase